FIX Correct `LDA` `fac` multiplier (#22696)
scikit-learn/scikit-learn: doc/whats_new/v1.1.rst

  edges slightly, which might result in a different output curve given the same
  data.

> - |Fix| :class:`discriminant_analysis.LinearDiscriminantAnalysis` now uses a
>   more correct variance-scaling coefficient which may result in different model

Reviewer: In this case, it's correct now and was wrong before?

```suggestion
- |Fix| :class:`discriminant_analysis.LinearDiscriminantAnalysis` now uses
  the correct variance-scaling coefficient which may result in different model
```

:mod:`sklearn.discriminant_analysis`
....................................

> - |Fix| :class:`discriminant_analysis.LinearDiscriminantAnalysis` now uses a
>   more correct variance-scaling coefficient which may result in different model

Reviewer: Same here:

```suggestion
- |Fix| :class:`discriminant_analysis.LinearDiscriminantAnalysis` now uses
  the correct variance-scaling coefficient which may result in different model
```
